ngi_pipeline/utils/pyutils.py

```diff
@@ -10,3 +11,23 @@ def flatten(nested_list):
                 yield sub
         else:
             yield elt
+
+def update_dict(original_dict, updater_dict):
```

Comment:
And it's my first shot at it so I may have missed something. It works for my config file.